### DISCUSSION TITLE
How to disable cursor

### DISCUSSION THREAD

**Discussion by piby180** [chakra-ui/chakra-ui#3428] [General]
Hey,

I am currently checking out Chakra UI and really liking it. One thing which I couldn't find any solution is this cursor issue. 

![image](https://user-images.githubusercontent.com/8469664/108681543-8e663200-74ef-11eb-8522-c05feece463e.png)

Cursor appears on multiple text/non-text components on mouse click even if they are non editable. How should I disable it?

Thanks!

**Comment by chr-ge** [comment]
I'm not sure I understand what you mean but you can display the `default` cursor instead of the `text` cursor by adding this prop: `cursor='default'`

**Reply by thedarshanjadhav** [reply]
cursor default is not working

**Reply by DawnSpark7** [reply]
The issue is due to caret browsing, not due to Chakra UI. Just press F7 it should disable the blinking.

**Reply by thedarshanjadhav** [reply]
ohk.. thank you!!
